apps/web/tests/utils/browser.ts

```diff
@@ -22,8 +22,8 @@ export async function initializeSession(page: Page, settings: ISessionOptions =
    */
   await page.addInitScript((currentSession) => {
     window.addEventListener('DOMContentLoaded', () => {
-      localStorage.setItem('auth_token', currentSession.token);
-      localStorage.setItem('novu_last_environment_id', currentSession.environment._id);
+      localStorage.setItem('nv_auth_token', currentSession.token);
+      localStorage.setItem('nv_last_environment_id', currentSession.environment._id);
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** Please use constants to avoid having to change in multiple places in future PRs‏

Reply:
In tests, I prefer verbosity over dryness as it makes test easy to debug and reason about. Given that the legacy constant will be removed very soon, I'd rather leave as is for now.